src/com/TTT/BoardPrinter.java

```diff
@@ -19,17 +19,31 @@ public void printBoard(Board board) {
         int rowSize = 3;
         int finalNewLine = (board.size() - 1);
 
-        for(int spaceIndex = 0; spaceIndex < board.size(); spaceIndex++) {
-            out.print(" " + board.markerAt(spaceIndex)+ " ");
+        for (int spaceIndex = 0; spaceIndex < board.size(); spaceIndex++) {
+            printSpace(board, spaceIndex);
 
-            if((spaceIndex + 1) % rowSize != 0)
+            if ((spaceIndex + 1) % rowSize != 0)
                 out.print("|");
 
-            if(separatorRows.contains(spaceIndex))
+            if (separatorRows.contains(spaceIndex))
                 out.print("\n===========\n");
 
-            if(spaceIndex == finalNewLine)
+            if (spaceIndex == finalNewLine)
                 out.print("\n");
         }
     }
-}
+
+    private void printSpace(Board board, int spaceIndex) {
+        String space = board.markerAt(spaceIndex);
+
+        if (space.equals("X")) {
```

Comment:
It feels like we have more than one way to represent a players mark. We are using string "X" and "O" here. We also have the enum `PlayerMakers.X/O`.  Can you think of a way to always use enum when referring to X/O. And colour it only when necessary?  

Think back to the Elixir TTT - if we asked you in future to change X and O to A and B, how many places would you need to update?